### PR TITLE
Login form added

### PR DESCRIPTION
**Pull request by renzowolox**
## Summary
- Login form added

## Screenshots
image

## Trello Card
https://trello.com/c/Xx3iBRnm/32-login#

### CODE REVIEW

File: maquetado/login.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body>
+  <main class="main">
+    <div class="login-content">
+      <img src="images/LogoWolox.png" alt="" class="login-img">
+      <form class="login-form" action="book-list.html" method="POST">
```
Comment (Dorian30) [inline maquetado/login.html]: I think there is no need to add the method here since we are only redirecting our page

File: maquetado/login.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        <div class="login-inputs-content">
```
Comment (Dorian30) [inline maquetado/login.html]: How about: 

```suggestion
        <div class="login-input-container">
```

File: maquetado/login.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+          <label class="login-label" for="email">Email</label>
+          <input type="text" class="login-input" name="email">
```
Comment (Dorian30) [inline maquetado/login.html]: You need to add an id to your inputs if you're going to use `for` attribute on your labels. 

The purpose of that attribute is to focus your input when you click on the corresponding label.

File: maquetado/login.html [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+        </div>
+        <div class="login-inputs-content">
+          <label class="login-label" for="password">Password</label>
+          <input type="password" class="login-input" name="password">
```
Comment (Dorian30) [inline maquetado/login.html]: Same

File: maquetado/login.html [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+        </div>
+        <div class="login-inputs-content">
+          <input type="submit" class="login-submit" value="Login">
+        </div>
```
Comment (Dorian30) [inline maquetado/login.html]: I think you don't need to wrap this inside a container 👀

File: maquetado/login.html [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+      </form>
+      <div class="sign-up-content">
+        <button class="sign-up">Sign Up</button>
+      </div>
```
Comment (Dorian30) [inline maquetado/login.html]: Same

File: maquetado/scss/login.scss [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+  max-width: 220px;
+  padding: 15px 18px;
+  width: 100%;
+}
+
+.login-img {
+  width: 80%;
```
Comment (Dorian30) [inline maquetado/scss/login.scss]: Use a fixed width.

File: maquetado/scss/login.scss [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+}
+
+.login-form {
+  margin-top: 15px;
```
Comment (Dorian30) [inline maquetado/scss/login.scss]: Avoid margin top and left for responsive reasons

File: maquetado/scss/login.scss [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+}
+
+.login-inputs-content {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 15px;
+  
+  &:last-child {
+    margin-bottom: 10px;
+  }
```
Comment (Dorian30) [inline maquetado/scss/login.scss]: Nice!

File: maquetado/scss/login.scss [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+}
+
+.login-label {
+  font-size: 11px;
```
Comment (leonardofrp5) [inline maquetado/scss/login.scss]: this font size you use a variable

File: maquetado/scss/login.scss [hunk 11]
```diff
@@ -0,0 +1,63 @@
...
+.login-input {
+  border-radius: 4px;
+  line-height: 1.7;
+}
+
+.login-submit {
+  background-color: $pear;
+  border-radius: 4px;
+  color: $white;
+  font-size: 11px;
```
Comment (leonardofrp5) [inline maquetado/scss/login.scss]: this font size you use a variable

File: maquetado/scss/login.scss [hunk 12]
```diff
@@ -0,0 +1,63 @@
...
+  border-top: 2px solid $nobel;
+  padding-top: 10px;
+  width: 100%;
+}
+
+.sign-up {
+  border: 2px solid $pear;
+  border-radius: 4px;
+  color: $pear;
+  font-size: 11px;
```
Comment (leonardofrp5) [inline maquetado/scss/login.scss]: this font size you use a variable

File: maquetado/login.html (hunk 1 above)
Comment (renzowolox) [reply maquetado/login.html]: Method removed.

File: maquetado/login.html (hunk 2 above)
Comment (renzowolox) [reply maquetado/login.html]: Class name changed

File: maquetado/login.html (hunk 3 above)
Comment (renzowolox) [reply maquetado/login.html]: Issue solved

File: maquetado/login.html (hunk 5 above)
Comment (renzowolox) [reply maquetado/login.html]: Div container removed

File: maquetado/login.html (hunk 6 above)
Comment (renzowolox) [reply maquetado/login.html]: Issue solved

File: maquetado/scss/login.scss (hunk 7 above)
Comment (renzowolox) [reply maquetado/scss/login.scss]: Width fixed

File: maquetado/scss/login.scss (hunk 8 above)
Comment (renzowolox) [reply maquetado/scss/login.scss]: Issue solved.

File: maquetado/scss/login.scss (hunk 10 above)
Comment (renzowolox) [reply maquetado/scss/login.scss]: Issue solved

File: maquetado/scss/login.scss (hunk 11 above)
Comment (renzowolox) [reply maquetado/scss/login.scss]: Issue solved.

File: maquetado/scss/login.scss (hunk 12 above)
Comment (renzowolox) [reply maquetado/scss/login.scss]: Issue solved.

File: maquetado/scss/login.scss [hunk 13]
```diff
@@ -0,0 +1,63 @@
...
+}
+
+.login-label {
+  font-size: $text-small;
+  margin-bottom: 7px;
+}
+
+.login-input {
+  border-radius: 4px;
+  line-height: 1.7;
```
Comment (leonardofrp5) [inline maquetado/scss/login.scss]: if it works like this? 1.7 and add only 1 ?

Comment (renzowolox) [reply maquetado/scss/login.scss]: With line-height: 1.7, the input have the same height that the inputs in the image
Instead with line-height, inputs are smaller.
*With line-height: 1*
image

*With line-height: 1.7*
image

**Pull request merged**